src/main/java/io/strimzi/kafka/bridge/AdminClientEndpoint.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+
+package io.strimzi.kafka.bridge;
+
+import io.strimzi.kafka.bridge.config.BridgeConfig;
+import io.strimzi.kafka.bridge.config.KafkaConfig;
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Handler;
+import io.vertx.core.Vertx;
+import io.vertx.kafka.admin.KafkaAdminClient;
+import io.vertx.kafka.admin.TopicDescription;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
+import java.util.Set;
+
+public abstract class AdminClientEndpoint implements BridgeEndpoint {
+    protected final Logger log = LoggerFactory.getLogger(getClass());
+
+    protected String name;
```

Comment:
It seems you do not write to the `name`.

Reply:
I added a line to set this value to "kafka-bridge-admin-client".